crates/router/src/connector/authorizedotnet/transformers.rs

```
@@ -1834,10 +1834,12 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsCompleteAuthorizeRouterD
             .as_ref()
             .and_then(|redirect_response| redirect_response.params.as_ref())
             .ok_or(errors::ConnectorError::ResponseDeserializationFailed)?;
-        let payer_id: Secret<String> =
-            serde_urlencoded::from_str::<PaypalQueryParams>(params.peek())
-                .change_context(errors::ConnectorError::ResponseDeserializationFailed)?
-                .payer_id;
+
+        let query_params: Option<PaypalQueryParams> = serde_urlencoded::from_str(params.peek())
+            .change_context(errors::ConnectorError::ResponseDeserializationFailed)
+            .ok();
```

why are we ignoring the error here?